modules/ROOT/pages/bloom-installation/bloom-prerequisites.adoc

```diff
@@ -63,3 +63,21 @@ For more information on supported versions of Neo4j, see link:https://support.ne
 ====
 Keep in mind that if the Bloom client is used with the server component, both client and server should be the same exact version.
 ====
+
+[[bloom-plugin-compatibility]]
+=== Bloom-plugin and Neo4j version compatibility
+
+[cols="<,<", options=header]
+|===
+| Bloom plugin version
+| Supported Neo4j version
+
+| 4.x
+| 4.x
+
+| 5.x
+| 5
```

Comment:
also in this case I would put 5.26.x (LTS), even if we should guarantee support also for 5.0 -> 5.26. So probably at lest 5.x should be mentioned

Reply:
Yep, updated!